packages/calendar/src/compute.js

```diff
@@ -380,6 +380,7 @@ export const bindDaysData = ({ days, data, colorScale, emptyColor }) => {
                 day.value = dayData.value
                 day.color = colorScale(dayData.value)
                 day.data = dayData
+                return
```

Comment:
What was this needed for?

Reply:
To break from the foreach loop. Since there isn't a duplicate day in a calendar, there is no need to evaluate the rest of the data after finding the right day.
This way we can save n² - (n²)/2 operations. At least, this is my undesrtanding.

Reply:
We can go further and put a binary search there instead of the foreach. This way instead of O(n²) this part would be O(nlogn). But i'm not sure if it is worth the extra lines for such small data size. It's not like we have millions of day in a single calendar component. The return statement was a simple one.